robottelo/ui/computeprofile.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
+from robottelo.ui.base import Base, UINoSuchElementError
+from robottelo.ui.locators import locators
+from robottelo.ui.navigator import Navigator
+
+
+class ComputeProfile(Base):
+    """Provides the CRUD functionality for Compute Profiles."""
+
+    def search(self, name):
+        """Search for existing compute profile from UI."""
+        Navigator(self.browser).go_to_compute_profiles()
+        self.wait_for_ajax()
+        element = self.search_entity(name, locators['profile.select_name'])
+        return element
+
+    def select_resource(self, profile_name, res_name, res_type):
+        """Select necessary compute resource from specific compute profile"""
+        profile = self.search(profile_name)
+        if profile is None:
+            raise UINoSuchElementError(
+                'Could not find the profile {0}'.format(profile_name))
```

Comment:
Please use unicode strings every time .format is called to avoid unicode encode errors.